docs/concepts/abstractions/overview.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Kubernetes Objects
+---
+
+{% capture overview %}
+This page explains how Kubernetes objects are represented in the Kubernetes API, and how you can express them in `.yaml` format.
+{% endcapture %}
+
+{% capture body %}
+### Understanding Kubernetes Objects
+
+*Kubernetes Objects* are persistent entities in the Kubernetes system. Kubenetes uses these entities to represent the state of your cluster. Specifically, they can describe:
+
+* What containerized applications are running (and on which nodes)
+* The resources available to those applications
+* The policies around how those applications behave, such as restart policies, upgrades, and fault-tolerance
+
+When you create a Kubernetes object, you create a "record of intent"--once you create the object, the Kubernetes system will constantly work to ensure that that object exists. By creating an object, you're effectively telling the Kubernetes system what you want your cluster's workload to look like; this is your cluster's **desired state**.
```

Comment:
Is this the same as saying: "The Kubernetes object is a record of intent - the Kubernetes..."